ophyd/controls/areadetector/detectors.py

```diff
@@ -1005,9 +1023,8 @@ def remove_macros(record):
     print('class %s(%s):' % (det_name, base_class.__name__))
 
     def get_prop_name(pv):
-        '''
-        A terribly confusing method to get a property name from
-        the camel-case AreaDetector PV names
+        '''A terribly confusing method to get a property name from the
+        camel-case AreaDetector PV names
```

Comment:
LOL: It also has areaDetector which also doesn’t inspire confidence …… :-)

On Jan 27, 2015, at 6:24 PM, Thomas A Caswell <notifications@github.com<mailto:notifications@github.com>> wrote:


In ophyd/controls/areadetector/detectors.py<https://github.com/NSLS-II/ophyd/pull/53#discussion_r23653107>:

> @@ -1005,9 +1023,8 @@ def remove_macros(record):
>      print('class %s(%s):' % (det_name, base_class.__name__))
>
>      def get_prop_name(pv):
> -        '''
> -        A terribly confusing method to get a property name from
> -        the camel-case AreaDetector PV names
> +        '''A terribly confusing method to get a property name from the
> +        camel-case AreaDetector PV names


This docstring does not inspire confidence in the function it documents.

—
Reply to this email directly or view it on GitHub<https://github.com/NSLS-II/ophyd/pull/53/files#r23653107>.